registry/serverless.yml

```diff
@@ -0,0 +1,243 @@
+    tags:
+      key1: registry
+    deploymentPrefix: serverless
+    tracing: true
+
+custom:
+  pythonRequirements:
+    fileName: contract_api/requirement.txt
+    dockerizePip: true
+    useDownloadCache: true
+    useStaticCache: true
+    cacheLocation: "/var/cache/serverless"
+  prune:
+    automatic: true
+    includeLayers: true
+    number: 5
+
+package:
+  exclude:
+    - .circleci/**
+    - .gitignore/**
+    - .serverless/**
+    - requirement.txt
+    - venv/**
+    - config.ropsten.json
+    - sls_deploy.sh
+    - serverless.yml
+    - test/**
+    - tests/**
+    - sql_script/**
+    - service_status/**
+    - dapp-user/**
+    - repository/**
+    - Readme.md
+    - parse_events.sh
+    - package.json
+    - Dockerfile
+    - License
+    - log_setup.py
+    - heath_check.sh
+    - node_modules/**
+    - wallets/**
+    - payments/**
+    - orchestrator/**
+    - notification/**
+    - dapp_user/**
+    - contract_api/**
+  include:
+    - node_modules/singularitynet-platform-contracts/**
+
+functions:
+  get_all_org:
+    handler: registry.application.handlers.organization_handlers.get_all_org
+    role: ${file(./config.${self:provider.stage}.json):ROLE}
+    vpc:
+      securityGroupIds:
+        - ${file(./config.${self:provider.stage}.json):SG1}
+        - ${file(./config.${self:provider.stage}.json):SG2}
+      subnetIds:
+        - ${file(./config.${self:provider.stage}.json):VPC1}
+        - ${file(./config.${self:provider.stage}.json):VPC2}
+    events:
+      - http:
+          method: GET
+          path: /org
+          cors:
+            origin: ${file(./config.${self:provider.stage}.json):ORIGIN}
+            headers:
+              - Content-Type
+              - X-Amz-Date
+              - Authorization
+              - X-Api-Key
+              - X-Amz-Security-Token
+              - X-Amz-User-Agent
+              - x-requested-with
+
+  add_org:
+    warmup: true
+    handler: registry.application.handlers.organization_handlers.add_org
+    role: ${file(./config.${self:provider.stage}.json):ROLE}
+    vpc:
+      securityGroupIds:
+        - ${file(./config.${self:provider.stage}.json):SG1}
+        - ${file(./config.${self:provider.stage}.json):SG2}
+      subnetIds:
+        - ${file(./config.${self:provider.stage}.json):VPC1}
+        - ${file(./config.${self:provider.stage}.json):VPC2}
+    events:
+      - http:
+          method: POST
+          path: /org
+          request:
+            parameters:
+              paths:
+                orgId: true
+          cors:
+            origin: ${file(./config.${self:provider.stage}.json):ORIGIN}
+            headers:
+              - Content-Type
+              - X-Amz-Date
+              - Authorization
+              - X-Api-Key
+              - X-Amz-Security-Token
+              - X-Amz-User-Agent
+              - x-requested-with
+
+  get_org:
+    warmup: true
+    handler: registry.application.handlers.organization_handlers.get_org
```

Comment:
ALso write the documentation/contract so that view can use the contract to desgin the frontend